### ISSUE TITLE
baseapi_test could not read images

### ISSUE THREAD

**Issue by Shreeshrii** [tesseract-ocr/tesseract#2517]
```
[----------] 2 tests from TesseractInstanceTest
[ RUN      ] TesseractInstanceTest.TestMultipleTessInstances
Could not read /home/ubuntu/tesseract/test/testing/HelloGoogle.tifCould not read /home/ubuntu/tesseract/test/testing/viet.tifCould not read /home/ubuntu/tesseract/test/testing/raaj.tifCould not read /home/ubuntu/tesseract/test/testing/arabic.tif[       OK ] TesseractInstanceTest.TestMultipleTessInstances (12792 ms)
[ RUN      ] TesseractInstanceTest.TestMultipleTessInstanceVariables
[       OK ] TesseractInstanceTest.TestMultipleTessInstanceVariables (197 ms)
[----------] 2 tests from TesseractInstanceTest (12990 ms total)
```

The files exist in the testing directory

```
 ls -l /home/ubuntu/tesseract/test/testing/*.tif
-rw-rw-r-- 1 ubuntu ubuntu     254 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/12.tif
-rw-rw-r-- 1 ubuntu ubuntu     272 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/136.tif
-rw-rw-r-- 1 ubuntu ubuntu     288 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/256.tif
-rw-rw-r-- 1 ubuntu ubuntu     290 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/324.tif
-rw-rw-r-- 1 ubuntu ubuntu     272 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/410.tif
-rw-rw-r-- 1 ubuntu ubuntu     280 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/432.tif
-rw-rw-r-- 1 ubuntu ubuntu     274 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/433.tif
-rw-rw-r-- 1 ubuntu ubuntu     276 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/540.tif
-rw-rw-r-- 1 ubuntu ubuntu     304 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/692.tif
-rw-rw-r-- 1 ubuntu ubuntu     286 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/779.tif
-rw-rw-r-- 1 ubuntu ubuntu     284 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/793.tif
-rw-rw-r-- 1 ubuntu ubuntu  112194 Mar  9 15:27 /home/ubuntu/tesseract/test/testing/8071_093.3B.tif
-rw-rw-r-- 1 ubuntu ubuntu   86066 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/8087_054.3B.tif
-rw-rw-r-- 1 ubuntu ubuntu 4228728 Mar  9 15:27 /home/ubuntu/tesseract/test/testing/8087_054.3G.tif
-rw-rw-r-- 1 ubuntu ubuntu     292 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/808.tif
-rw-rw-r-- 1 ubuntu ubuntu     274 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/815.tif
-rw-rw-r-- 1 ubuntu ubuntu     342 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/arabic.tif
-rw-rw-r-- 1 ubuntu ubuntu  122298 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/deslant.tif
-rw-rw-r-- 1 ubuntu ubuntu  102598 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/eurotext.tif
-rw-rw-r-- 1 ubuntu ubuntu     956 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/HelloGoogle.tif
-rw-rw-r-- 1 ubuntu ubuntu 1614184 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/phototest_2.tif
-rw-rw-r-- 1 ubuntu ubuntu   23604 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/phototestrot.tif
-rw-rw-r-- 1 ubuntu ubuntu   38668 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/phototest.tif
-rw-rw-r-- 1 ubuntu ubuntu     312 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/raaj.tif
-rw-rw-r-- 1 ubuntu ubuntu    1522 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/trainingitalline.tif
-rw-rw-r-- 1 ubuntu ubuntu    1522 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/trainingital.tif
-rw-rw-r-- 1 ubuntu ubuntu    1488 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/trainingtimesline.tif
-rw-rw-r-- 1 ubuntu ubuntu    1488 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/trainingtimes.tif
-rw-rw-r-- 1 ubuntu ubuntu     394 Jan 22 13:32 /home/ubuntu/tesseract/test/testing/viet.tif

```
and are found by other tests  in the same run. eg. `HOCRContainsBaseline` uses `HelloGoogle.tif` and passes.

```
  Pix* src_pix = pixRead(TestDataNameToPath("HelloGoogle.tif").c_str());
  CHECK(src_pix);
  api.SetInputName("HelloGoogle.tif");
```

```
[ RUN      ] TesseractTest.HOCRContainsBaseline
[       OK ] TesseractTest.HOCRContainsBaseline (186 ms)
```

 tesseract -v
tesseract 5.0.0-alpha-202-g9eb6
 leptonica-1.78.0
  libgif 5.1.4 : libjpeg 8d (libjpeg-turbo 1.4.2) : libpng 1.2.54 : libtiff 4.0.9 : zlib 1.2.11 : libwebp 0.4.4 : libopenjp2 2.3.0

**Comment by stweil** [comment]
Thank you for reporting this. The image was read successfully, but the test code had a wrong implementation of `QCHECK` which printed an error message for that. This is fixed in commit 252d80cb6dfdd68035d7de519204b6bb9a08bf82.

**Comment by Shreeshrii** [comment]
Thanks!
